### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by marziman**
Initial docu contribution

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,508 @@
...
+openHAB Cloud is mainly based on the following frameworks and technologies:
+
+
+* [Node.js](https://nodejs.org/en/) - Server-side Javascript-framework
+* [Express.js](http://redis.io) - Web application framework for Node.js
+* [Nginx](https://www.nginx.com/resources/wiki/) - Web server & reverse proxy
+* [MongoDB](https://www.mongodb.com/) - NoSQL database
+* [redis](http://redis.io) - Session Manager & data structure server
+* [Socket.IO](http://socket.io/) - Bi-directional communication between web clients and servers
+* [MEAN](https://en.wikipedia.org/wiki/MEAN_(software_bundle)) - Software stack
```
Comment (digitaldan) [inline README.md]: Is is MEAN, I don't think we have the "A' (angular) do we?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,508 @@
...
+First we need to make sure that the list of packages and dependencies
+from all repositories are up-to-date: 
+
+```
+sudo apt-get update
+sudo apt-get upgrade
+sudo apt-get dist-upgrade
+```
+
+If you got asked to continue, confirm with Yes (Y).
```
Comment (digitaldan) [inline README.md]: "If you are asked to continue" , I think you can also run these with "-y", alternatively we could leave this out, if you can't figure this out, you should not be installing openHAB Cloud ;-)

Comment (marziman) [inline README.md]: hehe ok.

File: README.md (hunk 1 above)
Comment (marziman) [inline README.md]: No we dont. Thats an old link, I will remove it.

**Pull request closed without merging**